frontend/public/components/resource-dropdown.tsx

```diff
@@ -65,7 +102,20 @@ const ResourceListDropdown_: React.SFC<ResourceListDropdownProps> = (props) => {
 
   React.useEffect(() => {
     setSelectedOptions(selected);
-  }, [selected]);
+    !_.isEmpty(selected) &&
+      setRecentSelected(
+        JSON.stringify(
+          _.union(filterGroupVersionKind(selected.reverse()), recentSelectedList(recentSelected)),
+        ),
+      );
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [selected, setRecentSelected]);
+
+  const onClear = () => {
+    setSelectedOptions([]);
+    clearAllFilters();
+    setRecentSelected(JSON.stringify([]));
+  };
```

Comment:
IMO, clear the resent used option should just do `setRecentSelected(JSON.stringify([]))`. It should not clear the slected options and clear all filters. cc: @jerolimov